### PR TITLE
Modify Introduction, OSS Use and Contribution

### PR DESCRIPTION
**Pull request by gcharest**
* Update Markets and Culture
* Review Introduction, Software Use and Contribution

### CODE REVIEW

File: en/6_Open_Culture.md [hunk 1]
```diff
@@ -3,13 +3,14 @@
 ## Open Culture
 
 - [Open Organization](#open-organization)
+- [Risks and Drawbacks](#risks-and-drawbacks)
```
Comment (nschonni) [inline en/6_Open_Culture.md]: This heading doesn't exist in this document

**Comment by gcharest** [comment]
Thanks, will be added shortly!

Also will be waiting on merge after discussing with @smellems regarding
French version update to match current modifications.

Le mer. 11 juill. 2018 18 h 18, Nick Schonning <notifications@github.com> a
écrit :

**Comment by gcharest** [comment]
Travis isn't happy... Build works on my box though, so...

**Comment by gcharest** [comment]
Still won't pass the build... :/

**Comment by gcharest** [comment]
Something is up with the server timing out on the links as it runs perfectly on my local machine.

**Pull request merged**